### DISCUSSION TITLE
Purge all data and database and do clean install

### DISCUSSION THREAD

**Discussion by etsiot** [photoprism/photoprism#3398] [Q&A] [answered]
Hi all, 

I have a fairly large library of photos and videos (100K+) so I first tried a test setup to check functionality, using docker compose on OSX.

As a second step I deleted the docker containers and the photoprism storage folder and started with a clean host folder and  docker compose.

To my surprise, when I logged in the UI in the fresh setup, before doing any indexing, the test folders, images etc were still there!
Only things missing were some thumbnails.

How is this possible? Are other folders being used in the originals folder?

**Comment by lastzero** [comment, marked as answer]
Note that the config example we provide for macOS uses a named Docker volume to store the database:

- https://dl.photoprism.app/docker/macos/docker-compose.yml

This is because I/O performance is very low when mounting host filesystems with Docker on macOS. If you delete the volume or use our reset CLI command, you will have a clean index.

**Reply by etsiot** [reply]
Thank you for that. I realised it myself earlier today, had not paid attention to the database volume definition. 
I was planning to create a bind volume but if performance sucks I will stick with what you have provided.

Have to say I am pretty impressed with what you have put together, very nice interface and cool functionality. 
Kudos, guys!
